compliance/locker.py

```diff
@@ -222,7 +222,8 @@ def add_evidence(self, evidence, checks=None, evidence_used=None):
                     f.write(evidence.get_partition(key))
         else:
             full_path = os.path.join(self.local_path, evidence.path)
```

Comment:
Using pathlib here might be nice. You can write o the Path, too.

Reply:
https://github.com/ComplianceAsCode/auditree-framework/issues/89

Reply:
Nice one, thanks